VSRAD.DebugServer/Server.cs

```diff
@@ -31,6 +35,14 @@ public Server(IPAddress ip, int port, bool verboseLogging = false)
             _verboseLogging = verboseLogging;
         }
 
+        public enum HandShakeStatus
+        {
+            CLIENT_ACCEPTED,
+            CLIENT_NOT_ACCEPTED,
+            SERVER_ACCEPTED,
+            SERVER_NOT_ACCEPTED
```

Comment:
To match our code style I would go with `ClientAccepted` etc

Reply:
done